src/app/search/loadouts/search-filters/freeform.ts

```diff
@@ -204,6 +305,113 @@ const freeformFilters: FilterDefinition<
       return (loadout) => test(loadout.name) || Boolean(loadout.notes && test(loadout.notes));
     },
   },
+  {
+    keywords: ['light', 'power'],
+    /* t('Filter.PowerKeywords') */
+    description: tl('LoadoutFilter.LoadoutLight'),
+    format: 'range',
+    overload: powerLevelByKeyword,
+    filter: ({ compare, allItems, d2Definitions, selectedLoadoutsStore }) => {
+      if (!d2Definitions || !selectedLoadoutsStore || !allItems) {
+        return () => false;
+      }
+      return (loadout: Loadout) => {
+        if (!isLoadoutCompatibleWithStore(loadout, selectedLoadoutsStore)) {
+          return false;
+        }
+        //
+        const resolvedLoadout = ResolveLoadoutToDimItems(
+          loadout,
+          d2Definitions,
+          allItems,
+          selectedLoadoutsStore,
+        );
+
+        // The UI, at the time of implementing this, only shows the light level using the following rules:
+        // 1. it only shows the light level if all armor + weapon slots have items assigned
+        // 2. it uses all weapons when calculating it (not just equipped)
```

Comment:
If this is true, it's a bug (I really regret allowing non-equipped items in loadouts)

Reply:
This seems to be true, and I can consistently reproduce it by adding a LL 1900 item onto a duplicate loadout. Sorry 🤣.